addons/binding/org.openhab.binding.nest/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,303 @@
+    </channel-type>
+
+    <channel-type id="LockedMinSetPoint" advanced="true">
+        <item-type>Number</item-type>
+        <label>Temperature</label>
+        <description>The locked range min set point in degrees Celsius</description>
+        <category>Temperature</category>
+        <state readOnly="false" min="18" max="42" />
+    </channel-type>
+
+    <channel-type id="Locked" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Locked</label>
+        <description>If the thermostat has the temperature locked to only be within a set range</description>
+        <state readOnly="false" />
+    </channel-type>
+
+    <channel-type id="Mode">
+        <item-type>String</item-type>
+        <label>Mode</label>
+        <description>Current mode of the Nest thermostat</description>
+        <state readOnly="false">
+            <options>
+                <option value="OFF">off</option>
+                <option value="HEAT">heating</option>
+                <option value="COOL">cooling</option>
+                <option value="HEATCOOL">heat/cool</option>
+            </options>
+        </state>
+    </channel-type>
+
+    <channel-type id="PreviousMode" advanced="true">
+        <item-type>String</item-type>
+        <label>Previous Mode</label>
+        <description>The previous mode of the Nest thermostat</description>
+        <state readOnly="true">
+            <options>
+                <option value="OFF">off</option>
+                <option value="HEAT">heating</option>
+                <option value="COOL">cooling</option>
+                <option value="HEATCOOL">heat/cool</option>
+            </options>
+        </state>
+    </channel-type>
+
+    <channel-type id="Humidity">
+        <item-type>Number</item-type>
+        <label>Humidity</label>
+        <description>Indicates the current relative humidity</description>
+        <category>Humidity</category>
+        <state pattern="%.1f" readOnly="true">
+        </state>
+    </channel-type>
+
+    <channel-type id="TimeToTarget">
+        <item-type>Number</item-type>
+        <label>Time To Target</label>
+        <description>Time left to the target temperature (mins) approximately</description>
+        <state readOnly="true">
+        </state>
+    </channel-type>
+
+    <channel-type id="CanHeat" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Can Heat</label>
+        <description>If the thermostat can actually turn on heating</description>
+        <state readOnly="true" />
+    </channel-type>
+
+    <channel-type id="CanCool" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Can Cool</label>
+        <description>If the thermostat can actually turn on cooling</description>
+        <state readOnly="true" />
+    </channel-type>
+
+    <channel-type id="HasFan" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Has Fan</label>
```

Comment:
@kaikreuzer maybe a stupid question but is it by design that there is no fallback to this label, if the channel which refers to this channel-type does not define its own channel

Example:

```
<channel id="has_fan" typeId="HasFan" />

<channel-type id="HasLeaf">
    <item-type>Switch</item-type>
    <label>Has Leaf</label>
    <description>If the thermostat is currently in a leaf mode</description>
    <state readOnly="true" />
</channel-type>
```

![image](https://user-images.githubusercontent.com/2625279/30446324-17a690b4-9989-11e7-8bdd-e8736fd777fd.png)

Reply:
No, it should definitely show the label from the channel type. If this is not the case, something is broken...
Are you testing that on latest ESH master? Do you have the same effect on other bindings?

Reply:
I'm running it from inside Eclipse so there are not that much other bindings, let me check the snapshot I'm running...

Reply:
I or someone else has to do more testing, my server is running in a different mode and does not automatically create the items when clicking on the link button, it has a popup requesting the label and that is properly prefilled. When running it from my IDE has has 'one-click-configuration' and there it seems to fail. I can try with another binding but looking at the upcoming days that will be no earlier than wednesday.

Reply:
I  see proper labels for the channels in Paper UI. I also don't see those sliders for values.